src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -102,7 +129,18 @@ const RunPipelineDialog = (props: RunPipelineDialogProps) => {
       for (const parameter of version.parameters) {
         const val = fields[parameter.code];
         if (parameter.type === "int") {
```

Comment:
@cheikhgwane I think we often said it but you do have to handle the case of the float as well.